app/controllers/publishers/site_banners_controller.rb

```diff
@@ -34,32 +16,35 @@ def fetch
     end
   end
 
-  def update_logo
-    if params[:image].length > MAX_IMAGE_SIZE
-      # (Albert Wang): We should consider supporting alerts. This might require a UI redesign
-      # alert[:error] = "File size too big!"
-      head :payload_too_large and return
+  def update
+    if site_banner
+      site_banner.update(publisher_id: current_publisher.id, title: sanitize(params[:title]), description: sanitize(params[:description]), donation_amounts: JSON.parse(sanitize(params[:donation_amounts])), default_donation: JSON.parse(sanitize(params[:donation_amounts])).second, social_links: params[:social_links].present? ? JSON.parse(sanitize(params[:social_links])) : {})
```

Comment:
<img src="https://media.giphy.com/media/3cB7aOM6347PW/giphy.gif" />

i'm normally a 120 characters line length kind of guy, but this is way too long even for me lol

Reply:
Let me backlog this one item, I agree we should extract out but I don't wanna break anything before deploy 👾